enhancements/okd-on-fedora-coreos.md

```diff
@@ -0,0 +1,76 @@
+---
+title: OKD on Fedora CoreOS
+authors:
+  - "@LorbusChris"
+reviewers:
+  - "@smarterclayton"
+  - "@miabbott"
+  - "@ashcrow"
+  - "@runcom"
+  - "@darkmuggle"
+  - "@vrutkovs"
+approvers:
+creation-date: 2019-10-21
+last-updated: 2019-10-21
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:  
+replaces:
+superseded-by:
+---
+
+# OKD-on-Fedora-CoreOS
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+ 1. This proposal potentially requires addition of non-Koji-built RPMs from OpenShift's Prow CI instance to Fedora CoreOS composes. Can we do this? If yes, how do we do this?
+ 2. What is the specific list of operators that will go into OKD-on-FCoS?
+
+ Please see the Graduation Criteria section for more context on both open questions.
+
+## Summary
+
+A proposal to create components of an OKD variant that runs atop Fedora CoreOS and to use OpenShift's Prow instance to do so.
+
+As part of the version 4 effort, the OKD working group has agreed to target Fedora CoreOS (FCoS) as the primary base operating system for builds of version 4 of the OKD Community Distribution that is powered by the OpenShift codebase.
+
+This document proposes to incrementally add Prow CI jobs to create Fedora-based artifacts for use in OKD version 4.
+
+## Motivation
+
+Delivering continuous builds of version 4 of OKD, based on Fedora CoreOS. 
```

Comment:
Overall makes sense to me. Just curious, will these end up as separate branches of the installer/operators if customization needs to happen?

Reply:
This is something I think we'll decide for each component as we go forward. The general directive is to try to keep everything in one canonical master, but at least installer and mco will have their own branches in the short/mid term, i.e. until we can coordinate merging those back into master.